packages/amplify-category-auth/src/provider-utils/awscloudformation/auth-stack-builder/auth-cognito-stack-builder.ts

```diff
@@ -646,6 +649,36 @@ export class AmplifyAuthCognitoStack extends cdk.Stack implements AmplifyAuthCog
     });
   }
 
+  /**
+   * creates OAuth customResource for Cognito
```

Comment:
We are not creating a `customResource`, right? But updating the Cognito's L1 construct with OAuth?

Reply:
yeah you are correct , the previous commit on this branch : https://github.com/aws-amplify/amplify-cli/commit/a391baf5cb0241052638e780a737691e08f4d9bf#diff-79c5ca11b4f77a8d94fb923276bd9beea861fa0d543c188627496b3b79939ad7 removes the OAuthcustom resource.

I can update the comment though